bridge/bridgeapp/forms.py

```diff
@@ -1,16 +1,18 @@
 from django import forms
 from bridgeapp.models import Category
-# from bridgeapp.models import Category, Thread, Response
 
-CAT_CHOICES = [(f"{category.id}", f"{category.type}") for category in Category.objects.all()]
+CAT_CHOICES = [(f"{category.id}", f"{category.type}")
+               for category in Category.objects.all()]
+
 
 class InputForm(forms.Form):
-    # body = forms.CharField(label='Add note', max_length=255)
-    body = forms.CharField(label='', widget=forms.Textarea(attrs={'rows': '2', 'cols': '50'}), required=True)
+    body = forms.CharField(label='', widget=forms.Textarea(
+        attrs={'rows': '2', 'cols': '50'}), required=True, )
+
 
 class ThreadForm(InputForm):
-  category_ids = forms.MultipleChoiceField(
-    label='Check applicable categories',
-    widget=forms.CheckboxSelectMultiple, choices=CAT_CHOICES,
-    required=True
+    category_ids = forms.MultipleChoiceField(
+        label='Check applicable categories',
+        widget=forms.CheckboxSelectMultiple, choices=CAT_CHOICES,
```

Comment:
Should be
```
widget=forms.CheckboxSelectMultiple,
choices=CAT_CHOICES,
```
if you want to be stylistically consistent, though I see it might be the software that did the folding.
Also, add an attribute to the widget will default-check all the boxes and require the user to uncheck inapplicable ones, to prevent inadvertent submission without selecting _at least one_ catergory:
```
label='Uncheck inapplicable categories',
widget=forms.CheckboxSelectMultiple(attrs={'checked': True}),
choices=CAT_CHOICES,
```

Reply:
You're right, the stylistic changes were implemented automatically, not done by me. I was planning to do the check boxes as a separate issue